mb-solr/src/main/java/org/musicbrainz/search/analysis/MusicbrainzTokenizer.java

```diff
@@ -0,0 +1,176 @@
+/*
```

Comment:
The licene for most other files in this repo is MIT, this is Apache 2.0 code. This needs to either be documented somewhere or changed with permission of the authors.